.github/workflows/server_compatibility.yaml

```diff
@@ -909,12 +909,29 @@ jobs:
         run: |
           echo "HZ_LICENSEKEY=${{ secrets.HAZELCAST_ENTERPRISE_KEY_V7 }}" >> $GITHUB_ENV
 
+      - name: Build modules
+        shell: bash -l {0}
+        run: |
+          chmod +x mvnw
+          ./mvnw -B -V -e clean install -DskipTests
+        working-directory: client
+
       - name: Run non-enterprise tests
+        if: ${{ matrix.server_kind == 'os' }}
```

Comment:
os scoped tests also should run against ee server.

Reply:
I added them as a separate job